weasyl/define.py

```diff
@@ -319,28 +319,47 @@ def _get_csrf_input():
 
 @region.cache_on_arguments(namespace='v2')
```

Comment:
Since the return format is changing, this should be updated to `v3` so it’s not necessary to clear memcached to avoid errors.